test/parallel/test-stream-pipeline.js

```diff
@@ -1048,3 +1049,43 @@ const { promisify } = require('util');
   }));
   src.push(null);
 }
+
+{
+  let closed = false;
+  const src = new Readable({
+    read() {},
+    destroy(err, cb) {
+      process.nextTick(cb);
+    }
+  });
+  const dst = new Writable({
+    write(chunk, encoding, callback) {
+      callback();
+    }
+  });
+  src.on('close', () => {
+    closed = true;
+  });
+  src.push(null);
+  pipeline(src, dst, common.mustCall((err) => {
+    assert.strictEqual(closed, true);
+  }));
+}
+
+{
+  let closed = false;
+  const src = new Readable({
+    read() {},
+    destroy(err, cb) {
+      process.nextTick(cb);
+    }
+  });
+  const dst = new Duplex({});
+  src.on('close', () => {
```

Comment:
Why not use `common.mustCall()` here?

Reply:
`assert.strictEqual(closed, true);` can't be true unless it has been called... though I guess it wouldn't hurt